packages/styles/src/popover.scss

```diff
@@ -6,13 +6,31 @@
 $block: #{$fd-namespace}-popover;
 $blockBody: #{$fd-namespace}-popover__body;
 
+body.overlay-active::before {
```

Comment:
`overlay-active` is not a proper name. If the application that uses Fundamental-styles also uses another library and they have a class called `overlay-active` there will be unexpected behaviour. Your class should be prefixed with `fd-` to avoid this